apps/cowswap-frontend/src/modules/usdAmount/updaters/UsdPricesUpdater.ts

```diff
@@ -95,9 +95,10 @@ async function processQueue(queue: Token[], getUsdcPrice: () => Promise<Fraction
         if (price) {
           state.price = price
           state.updatedAt = Date.now()
+          console.debug(`[UsdPricesUpdater]: Fetched price for`, currency.symbol, price?.toFixed(6))
```

Comment:
Should be deleted before merging

Reply:
Yup, it's on the description 
![image](https://github.com/cowprotocol/cowswap/assets/43217/7348ee8d-66bc-493e-853f-7103f2f11f91)